parent/pom.xml

```diff
@@ -593,6 +593,12 @@
                 <artifactId>onnxruntime</artifactId>
                 <version>${onnxruntime.vespa.version}</version>
             </dependency>
+           <dependency>
+               <groupId>de.kherud</groupId>
+               <artifactId>llama</artifactId>
+               <version>${java-llama.vespa.version}</version>
+               <scope>compile</scope>
```

Comment:
Please remove scope from the dependencyManagement. It should contain version only.